src/main/resources/db/migration/V1_17__add_and_drop_indexes_on_errand_tabel.sql

```diff
@@ -0,0 +1,9 @@
+create index if not exists idx_errand_municipality_id_namespace_status_created
+    on errand (municipality_id,namespace,status,created);
+
+create index if not exists idx_errand_municipality_id_namespace_status_touched
+    on errand (municipality_id,namespace,status,touched);
+
```

Comment:
drop "idx_errand_municipality_id_namespace_modified" as well?

Reply:
There is no index "idx_errand_municipality_id_namespace_modified". It is a diff between flyway and annotations. This is a correction of annotations.